flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "only-arches": ["x86_64"]
```

Comment:
why x86_64 only?

Reply:
Because the upstream did not provide the support of aarch64 in Linux

Reply:
do they support flatpak?

Reply:
Yes, they all support them, and they are all graphics applications

Reply:
so why not build aarch64 then?

Reply:
The build target in the build script provided upstream does not have aarch64, only arm

Reply:
This is the supported targets https://www.gryphel.com/c/minivmac/options.html#option_t